HW_16/task_3.py

```diff
@@ -0,0 +1,65 @@
+class Product:
+    def __init__(self, type_, name, price):
+        self.type_ = type_
+        self.name = name
+        self.price = price
+
+    def __str__(self):
+        pass
+
+class ProductStore:
+    def __init__(self):
+        self.warehouse = {}
+        self.income = 0
+
+    def add(self, product: object, amount: int):
+        product.price = product.price * 1.3
+        self.warehouse[product.name] = {
+        "type_": product.type_,
+        "price": product.price,
+        "amount": amount
+        }
+
+
+    def set_discount(self, identifier, percent, identifier_type="name"):
+        for key, value in self.warehouse.items():
+            if identifier == value["type_"] or identifier_type == key:
```

Comment:
I understand the logic behind `identifier_type == key`, but I don't get the `identifier == value["type_"]`. In this case `identifier_type` must also be `type`